src/constants/poi.data.ts

```diff
@@ -1735,19 +1760,6 @@ export const POIInfo: POI[] = [
     level: 1,
     category: POICategory.Exit
   },
-  {
-    id: "7fc86bc6-bc8e-4d2e-a8b0-e6b08d0cdcd9",
-    displayName: "Police St. Exit",
-
-    description: "Exit/Entrance to JMSB and the corner of Police St. and Guy",
-    location: {
-      latitude: 45.495216991097614,
-      longitude: -73.5785603480603
-    },
-    buildingId: BuildingId.MB,
-    level: 1,
-    category: POICategory.Exit
-  },
```

Comment:
Why was this removed?

Reply:
We agreed to have only 1 exit per building to simplify indoor navigation